backend/codesirius/codesirius/settings.py

```
@@ -31,7 +31,14 @@
 DEBUG = environ.get("DJANGO_DEBUG", "False") == "True"
 
 
-ALLOWED_HOSTS = ["127.0.0.1", "0.0.0.0", "localhost", "backend"]
+ALLOWED_HOSTS = [
+    "127.0.0.1",
+    "0.0.0.0",
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)